### DISCUSSION TITLE
What about a Gitter instance?

### DISCUSSION THREAD

**Discussion by wonea** [opengaming/osgameclones#1382] [General]
I was playing around with Gitter and was wondering whether osgameclones could make use of it?

Example
https://gitter.im/tagspaces/tagspaces#

**Comment by sansolovyov** [comment]
Ha-ha, but what would be discussed there? :) Anyway, I'm not against making a Gitter room, but I don't think I'll have enough time to actively participate there... :\

**Comment by wonea** [comment]
To be honest, I'm not entirely sure what will be discussed there. Guess I'm a little curious after seeing the activity on http://forum.freegamedev.net/

**Comment by cxong** [comment]
Yeah this sites community is too small...
I think we should join a bigger one if it exists, say some discord server or the like
